aiohttp/web_urldispatcher.py

```diff
@@ -353,24 +401,29 @@ def register_route(self, route):
                 self._routes[name] = route
         self._urls.append(route)
 
-    def add_route(self, method, path, handler,
-                  *, name=None, expect_handler=None):
-
-        if not path.startswith('/'):
-            raise ValueError("path should be started with /")
+    def add_view(self, method, handler, *, name='', route=None):
```

Comment:
Hm, I was thought that `route` argument should take Route instance, not the name before read the tests. Does it make same to handle both here?